tensorzero-core/src/config_parser/mod.rs

```diff
@@ -135,6 +135,66 @@ pub struct GatewayConfig {
     pub unstable_error_json: bool,
 }
 
+impl UninitializedGatewayConfig {
```

Comment:
could we pull the gateway config logic into a submodule? (just a file). This file is huge already.